specfile/changelog.py

```diff
@@ -83,25 +100,24 @@ def __repr__(self) -> str:
         return f"ChangelogEntry({self.header!r}, {self.content!r}, {self._following_lines!r})"
 
     @property
-    def evr(self) -> Optional[str]:
+    def evr(self) -> Optional[EVR]:
         """EVR (epoch, version, release) of the entry."""
-        m = re.match(
-            r"""
-            ^.*
-            \s+                       # preceding whitespace
-            ((?P<sb>\[)|(?P<rb>\())?  # optional opening bracket
-            (?P<evr>(\d+:)?\S+-\S+?)  # EVR
-            (?(sb)\]|(?(rb)\)))       # matching closing bracket
-            :?                        # optional colon
-            \s*                       # optional following whitespace
-            $
-            """,
-            self.header,
-            re.VERBOSE,
-        )
+        m = self._EVR_RE.match(self.header)
+
         if not m:
             return None
-        return m.group("evr")
+
+        evr_s = m.group("evr")
+        try:
+            evr = EVR.from_string(evr_s)
+        except SpecfileException:
+            return None
+
+        # looks like an email
+        if evr_s.startswith("<") and evr_s.endswith(">"):
+            return None
+
```

Comment:
I was thinking, if we get here, we could also check if the string contains a `@` or a number, EVR is not likely to contain a `@`  and on the other hand is likely to contain a number, unlike a surname for example. But maybe that's too much guessing.

Reply:
I wouldn't check for an `@`, some people obscure their emails like this one: `mszpak ATT wp DOTT pl`